edn/src/types.rs

```diff
@@ -563,7 +563,7 @@ pub trait FromMicros {
 
 impl FromMicros for DateTime<UTC> {
     fn from_micros(ts: i64) -> Self {
-        UTC.timestamp(ts / 100_000, ((ts % 100_000).abs() as u32) * 1_000)
+        UTC.timestamp(ts / 1_000_000, ((ts % 1_000_000).abs() as u32) * 1_000)
```

Comment:
Is it worth defining a constant for `1_000_000` that we can use here so if we need to implement `FromMicros` for another type we are less likely to make the same mistake?